src/app/address_separator/utils/extract/city.py

```diff
@@ -1,53 +1,53 @@
 from __future__ import annotations
 
 
-def extract_city(non_prefecture_address_data: list[str]) -> tuple[list[str], list[str]]:
+def extract_city(non_prefecture_address_data: str) -> tuple[str, str]:
     """
     市、区、群、町、村の順で文字列を探索し、分割する。
     if文の順番が 市->区->郡->町->村であるのは、分割パターンの都合上最もうまくいくように
     """
-    cities: list[str] = []
-    non_city_address_data: list[str] = []
+    cities: str = ""
+    non_city_address_data: str = ""
 
-    for city_address_data in non_prefecture_address_data:
-        if "市" in city_address_data:
-            index: int = city_address_data.find("市")
-            cities.append(city_address_data[0 : index + 1])
-            if len(city_address_data) > index + 1:
-                # avoid index out of range error
-                non_city_address_data.append(city_address_data[index + 1 :])
-            else:
-                non_city_address_data.append("")
-        elif "区" in city_address_data:
-            index: int = city_address_data.find("区")
-            cities.append(city_address_data[0 : index + 1])
-            if len(city_address_data) > index + 1:
-                non_city_address_data.append(city_address_data[index + 1 :])
-            else:
-                non_city_address_data.append("")
-        elif "郡" in city_address_data:
-            index: int = city_address_data.find("郡")
-            cities.append(city_address_data[0 : index + 1])
-            if len(city_address_data) > index + 1:
-                non_city_address_data.append(city_address_data[index + 1 :])
-            else:
-                non_city_address_data.append("")
-        elif "町" in city_address_data:
-            index: int = city_address_data.find("町")
-            cities.append(city_address_data[0 : index + 1])
-            if len(city_address_data) > index + 1:
-                non_city_address_data.append(city_address_data[index + 1 :])
-            else:
-                non_city_address_data.append("")
-        elif "村" in city_address_data:
-            index: int = city_address_data.find("村")
-            cities.append(city_address_data[0 : index + 1])
-            if len(city_address_data) > index + 1:
-                non_city_address_data.append(city_address_data[index + 1 :])
-            else:
-                non_city_address_data.append("")
+    city_address_data = non_prefecture_address_data
```

Comment:
```suggestion
    city_address_data: str = non_prefecture_address_data
```
変数宣言なので、type annotationを書いた方が良さそうです。
静的型付け言語の経験がないかもなので、詳しくは城戸さんに聞くか、調べるかしていただきたいのですが、変数宣言時に型を明示する言語が多いのでその影響を受けています。

Pythonがメイン言語の開発者にとっては慣れないかもしれませんが、何卒